### ISSUE TITLE
Failure to compile netlib with MSVC

### ISSUE THREAD

**Issue by mathstuf** [mozilla/sccache#802]
This is code embedded into ITK, but these failures show up:

```
FAILED: Modules/ThirdParty/VNL/src/vxl/v3p/netlib/CMakeFiles/itkv3p_netlib.dir/blas/cdotc.c.obj 
sccache C:\PROGRA~2\MICROS~1\2019\COMMUN~1\VC\Tools\MSVC\1426~1.288\bin\Hostx64\x64\cl.exe  /nologo -D /MP  -DV3P_NETLIB_SRC -DVXL_WARN_DEPRECATED -DVXL_WARN_DEPRECATED_ONCE -D_CRT_FAR_MAPPINGS_NO_DEPRECATE -D_CRT_IS_WCTYPE_NO_DEPRECATE -D_CRT_MANAGED_FP_NO_DEPRECATE -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_DEPRECATE_GLOBALS -D_CRT_SETERRORMODE_BEEP_SLEEP_NO_DEPRECATE -D_CRT_TIME_FUNCTIONS_NO_DEPRECATE -D_CRT_VCCLRIT_NO_DEPRECATE -D_SCL_SECURE_NO_DEPRECATE -IC:\GLR\builds\cmb\cmb-ci\build\superbuild\itk\src\Modules\ThirdParty\VNL\src\vxl\v3p\netlib -IC:\GLR\builds\cmb\cmb-ci\build\superbuild\itk\src\Modules\ThirdParty\VNL\src\vxl\vcl -IC:\GLR\builds\cmb\cmb-ci\build\superbuild\itk\src\Modules\ThirdParty\VNL\src\vxl\core -IModules\ThirdParty\VNL\src\vxl\v3p\netlib -IModules\ThirdParty\VNL\src\vxl\vcl -IModules\ThirdParty\VNL\src\vxl\core /DWIN32 /D_WINDOWS   /W0  /MD /O2 /Ob2 /DNDEBUG   /FS /showIncludes /FoModules\ThirdParty\VNL\src\vxl\v3p\netlib\CMakeFiles\itkv3p_netlib.dir\blas\cdotc.c.obj /FdModules\ThirdParty\VNL\src\vxl\v3p\netlib\CMakeFiles\itkv3p_netlib.dir\itkv3p_netlib.pdb /FS -c C:\GLR\builds\cmb\cmb-ci\build\superbuild\itk\src\Modules\ThirdParty\VNL\src\vxl\v3p\netlib\blas\cdotc.c
C:\GLR\builds\cmb\cmb-ci\build\superbuild\itk\src\Modules\ThirdParty\VNL\src\vxl\v3p\netlib\blas\cdotc.c(19): error C2061: syntax error: identifier 'cdotc_'
C:\GLR\builds\cmb\cmb-ci\build\superbuild\itk\src\Modules\ThirdParty\VNL\src\vxl\v3p\netlib\blas\cdotc.c(19): error C2059: syntax error: ';'
C:\GLR\builds\cmb\cmb-ci\build\superbuild\itk\src\Modules\ThirdParty\VNL\src\vxl\v3p\netlib\blas\cdotc.c(19): error C2143: syntax error: missing ')' before '*'
C:\GLR\builds\cmb\cmb-ci\build\superbuild\itk\src\Modules\ThirdParty\VNL\src\vxl\v3p\netlib\blas\cdotc.c(19): error C2143: syntax error: missing '{' before '*'
C:\GLR\builds\cmb\cmb-ci\build\superbuild\itk\src\Modules\ThirdParty\VNL\src\vxl\v3p\netlib\blas\cdotc.c(19): error C2143: syntax error: missing ';' before '*'
C:\GLR\builds\cmb\cmb-ci\build\superbuild\itk\src\Modules\ThirdParty\VNL\src\vxl\v3p\netlib\blas\cdotc.c(20): error C2143: syntax error: missing ';' before '*'
C:\GLR\builds\cmb\cmb-ci\build\superbuild\itk\src\Modules\ThirdParty\VNL\src\vxl\v3p\netlib\blas\cdotc.c(20): error C2059: syntax error: ')'
C:\GLR\builds\cmb\cmb-ci\build\superbuild\itk\src\Modules\ThirdParty\VNL\src\vxl\v3p\netlib\blas\cdotc.c(21): error C2054: expected '(' to follow 'incy'
```

It looks like some preprocessor expansion failure. I'll be looking into it locally, but wanted to file an issue in case anyone else has insights before I go down the wrong rabbit holes :) .

The relevant code is here: https://github.com/InsightSoftwareConsortium/ITK/tree/master/Modules/ThirdParty/VNL/src/vxl/v3p/netlib/blas

**Comment by froydnj** [comment]
Is this fixed by #801 ?

**Comment by mathstuf** [comment]
Ha, nice timing :) . I'll have to get a Windows build to test it out…looking into that.

**Comment by Dushistov** [comment]
May be this is the reason why there are no prebuild for release for windows for version 2.2.13,
but for 2.2.12 and bellow there are pre-build version  for windows.

**Comment by froydnj** [comment]
Fixed by #801. 

> May be this is the reason why there are no prebuild for release for windows for version 2.2.13,
> but for 2.2.12 and bellow there are pre-build version for windows.

No, this is not the reason for that.
